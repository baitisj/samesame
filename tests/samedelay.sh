NAME=samedelay
. ${srcdir}/tests/include_action.sh
