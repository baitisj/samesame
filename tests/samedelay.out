samedelay -A:
11	./work/test1/file1a	./work/test1/file2a	=	2	3
11	./work/test1/file1a	./work/test1/file3a	=	2	1

samedelay -Z:
11	./work/test1/file3a	./work/test1/file2a	=	1	3
11	./work/test1/file3a	./work/test1/file1a	=	1	2

samedelay -At:
11	./work/test1/file1a	./work/test1/file3a	=	2	1
11	./work/test1/file1a	./work/test1/file2a	=	2	3

samedelay -Zt:
11	./work/test1/file2a	./work/test1/file3a	=	3	1
11	./work/test1/file2a	./work/test1/file1a	=	3	2

samedelay -L:
11	./work/test1/file2a	./work/test1/file1a	=	3	2
11	./work/test1/file2a	./work/test1/file3a	=	3	1
